releasetool/commands/publish_reporter.py

```diff
@@ -21,6 +21,39 @@
 import releasetool.github
 
 
+def figure_out_github_token(github_token):
```

Comment:
There's type annotations on the other functions and not on this one. :upside_down_face:

Reply:
Good catch. We should also get around to actually enabling mypy here. :(